tiup: give a more clear recommendation about SELinux (#20068)
pingcap/docs: check-before-deployment.md

## Disable SELinux

> Check whether SELinux is enabled. To check the current status use the [getenforce(8)](https://linux.die.net/man/8/getenforce) utility. It is required to disable SELinux or have it in Permissive mode.

Reviewer: ```suggestion
SELinux must be disabled or set to permissive mode. To check the current status, use the [getenforce(8)](https://linux.die.net/man/8/getenforce) utility.
```

> If SELinux is not disabled, open `/etc/selinux/config`, locate the line starting with `SELINUX=`, and change it to `SELINUX=disabled`. After making this change, you need to reboot the system because switching from `enforcing` or `permissive` to `disabled` does not take effect without a reboot.

Reviewer: ```suggestion
If SELinux is not disabled, open the `/etc/selinux/config` file, locate the line starting with `SELINUX=`, and change it to `SELINUX=disabled`. After making this change, you need to reboot the system because switching from `enforcing` or `permissive` to `disabled` does not take effect without a reboot.
```

> On some systems (e.g. Ubuntu) the `/etc/selinux/config` file might not exist and the `getenforce` command might not be installed, in that case you can skip this step.
> 
> If SELinux is not disabled, change the line in `/etc/selinux/config` that starts with `SELINUX=` to have it say `SELINUX=disabled`. After changing this line you need to reboot the system as SELinux doesn't allow you to change the mode from Enforcing or Permissive to Disabled without a reboot.

Reviewer: ```suggestion
If SELinux is not disabled, open `/etc/selinux/config`, locate the line starting with `SELINUX=`, and change it to `SELINUX=disabled`. After making this change, you need to reboot the system because switching from `enforcing` or `permissive` to `disabled` does not take effect without a reboot.

On some systems (such as Ubuntu), the `/etc/selinux/config` file might not exist, and the `getenforce` might not be installed. In that case, you can skip this step.
```

Author: I think "the getenforce" should be changed to "getenforce" or "the getenforce utility".

Reviewer: Done.
